file_store/src/file_sink.rs

```diff
@@ -123,6 +123,7 @@ impl FileSinkBuilder {
         };
 
         metrics::register_counter!(client.metric);
+        metrics::register_counter!(client.metric, vec![OK_LABEL]);
```

Comment:
does this need both or can you just tack the `vec![OK_LABEL]` on to the existing counter registration? won't this just end up with two metrics registered, one with and one without the label as-is?

Reply:
yes, isn't that want we wanted?

Reply:
maybe it doesn't matter? i thought we more wanted a single metric that tracks the data across restarts. this feels like it will just leave a second dangling counter metrics _without_ the label that never gets used or incremented

Reply:
so just get rid of `  metrics::register_counter!(client.metric);`??

Reply:
That’s what I thought

Reply:
Removed 125aa52